src/test/apps/process-transition-test/App/wwwroot/login.html

```
@@ -0,0 +1,2 @@
+<h2>Placeholder page for Cypress to set origin before logging in via Tenor</h2>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)